src/aggregator/server/rawtcp/server.go

```diff
@@ -142,30 +142,37 @@ func (s *handler) Handle(conn net.Conn) {
 		switch current.Type {
 		case encoding.CounterWithMetadatasType:
 			untimedMetric = current.CounterWithMetadatas.Counter.ToUnion()
+			untimedMetric.Annotation = current.CounterWithMetadatas.Annotation
```

Comment:
is this intentional? `.ToUnion()` already sets the annotation

Reply:
Hi Vytenis. Thanks for reviewing it. Most likely you are right. I'll check it later and do a change if needed.

Reply:
no worries, was touching this code and noticed something new - thanks for confirming.

Reply:
actually nevermind - this needs a better look, I don't think annotations are set at all for unaggregated _untimed_ payloads